atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -135,6 +135,13 @@ def set_help(self, extra, worker_metadatas):
                         # They are all None
                         extra['image']['help'] = None
 
+    def set_media_types(self, extra, worker_metadatas):
+        for platform in worker_metadatas:
+            annotations = get_worker_build_info(self.workflow, platform).build.get_annotations()
+            if 'media-types' in annotations and annotations['media-types']:
```

Comment:
This can be simplified as: `if annotations.get('media-types'):`
The `.get` dict method will return a default value, `None` in this case, if key is not set.